tests/test_utils.py

```diff
@@ -964,7 +970,7 @@ def test_crop_and_pad_edges_non_positive_bounds_size():
 
 def test_normalise_padding_input_dims():
     """Test that normalise padding error with input dimensions > 1."""
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError, match="1 dimensional"):
         utils.image.normalise_padding_size(((0, 0), (0, 0)))
 
```

Comment:
Will crash, need to change `"1 dimensional"` to dimensional/dimentional either in `normalise_padding_size` or in pytest.